install/kubernetes/cilium/values.yaml

```diff
@@ -2050,6 +2050,10 @@ proxy:
 envoy:
   # -- Enable Envoy Proxy in standalone DaemonSet.
   enabled: false
+  # -- (int) Set Envoy --base-id to use when allocating shared memory regions.
+  # -- Only needs to be changed if multiple Envoy instances will run on the same node and may have conflicts.
+  # -- Defaults to '0'
+  baseID: 0
```

Comment:
Oh, I missed this. Please apply these changes to the `values.yaml.tmpl` and then run `make -C install/kubernetes`. This file here is auto-generated.